notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -123,7 +141,14 @@ define([
 
     KeyboardManager.prototype.bind_events = function () {
         var that = this;
+        var notebook_view = window.document.getElementById("pager");
```

Comment:
This doesn't seem to be a very robust way to test whether we are in notebook or tree view (a change to the HTML will break this logic). Is there a better way to test for this?

If not, please add a comment that this test expects "pager" to only exists when in notebook view.